integration/out_test.go

```diff
@@ -463,4 +482,93 @@ var _ = Describe("out", func() {
 			})
 		})
 	})
+	Describe("with content_type", func() {
+		var (
+			outRequest  out.OutRequest
+			filePath    string
+			tarballName string
+		)
+
+		Context("upload the pivotal file with application/octet-stream and get *.pivotal back", func() {
+			BeforeEach(func() {
+				// upload the .pivotal file
+				filePath = filepath.Join(sourceDir, "file-to-upload")
+				tarballName = "output-success.pivotal"
+
+				err = ioutil.WriteFile(filePath, []byte("contents"), 0755)
+				Expect(err).ToNot(HaveOccurred())
+
+				err = archiver.Zip.Make(filepath.Join(sourceDir, tarballName), []string{filePath})
+
+				// upload file with cmd
+				outRequest = out.OutRequest{
+					Source: gcsresource.Source{
+						JSONKey: jsonKey,
+						Bucket:  bucketName,
+						Regexp:  tarballName,
+					},
+					Params: out.Params{
+						File:        tarballName,
+						ContentType: "application/octet-stream",
+					},
+				}
+
+				err = json.NewEncoder(stdin).Encode(outRequest)
+				Expect(err).ToNot(HaveOccurred())
+
+			})
+			AfterEach(func() {
+				err := gcsClient.DeleteObject(bucketName, tarballName, int64(0))
+				Expect(err).ToNot(HaveOccurred())
+			})
+
+			It("the content-type of file should be application/octet-stream", func() {
+				object, err := gcsClient.GetBucketObjectInfo(bucketName, tarballName)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(object.ContentType).To(Equal("application/octet-stream"))
+			})
+
+		})
+
+		Context("upload the zip file with empty content type and get zip file back", func() {
+			BeforeEach(func() {
+				// upload the .pivotal file
+				filePath = filepath.Join(sourceDir, "file-to-upload")
+				tarballName = "output-fail.pivotal"
+				err = ioutil.WriteFile(filePath, []byte("contents"), 0755)
+				Expect(err).ToNot(HaveOccurred())
+
+				err = archiver.Zip.Make(filepath.Join(sourceDir, tarballName), []string{filePath})
+				Expect(err).ToNot(HaveOccurred())
+
+				// upload file with cmd
+				outRequest = out.OutRequest{
+					Source: gcsresource.Source{
+						JSONKey: jsonKey,
+						Bucket:  bucketName,
+						Regexp:  tarballName,
```

Comment:
Can you please use a `directoryPrefix` (like the other tests)? Otherwise, when running tests in parallel, collisions might occur, and the test GCS bucket gets polluted.